src/circleci/rollcage/core.clj

```diff
@@ -187,3 +188,24 @@
 (def warning  (partial notify "warning"))
 (def info     (partial notify "info"))
 (def debug    (partial notify "debug"))
+
+(defn rollbar-reporter
```

Comment:
If we move the try/catch code into `send-item` then we don't need `rollbar-reporter` - since it can then be replaced with `(partial rollcage/notify "error" client)`.